sklearn/neural_network/_multilayer_perceptron.py

```diff
@@ -1163,8 +1165,8 @@ def predict(self, X):
 
         return self._label_binarizer.inverse_transform(y_pred)
 
-    @property
-    def partial_fit(self):
+    @available_if(lambda est: est._check_solver())
```

Comment:
I like this approach of using a `lambda` to call into the parent's method. Let's see what other reviewers think.

Reply:
It is fine with me as well